### DISCUSSION TITLE
Lossless video support

### DISCUSSION THREAD

**Discussion by danielkoohmarey-bc** [aiortc/aiortc#617] [Q&A] [answered]
Is there any lossless video support though aiortc, if so how would one configure it? I see there are some flags for ffmpeg h.264 that might support, meaning av codec might have an option for it.

**Comment by jlaine** [comment, marked as answer]
What do you mean by "lossless video", what codec specifically are you referring to? I'm not aware of any "lossless" codecs standardized for use in WebRTC as it would be prohibitively expensive in bandwidth.

**Reply by danielkoohmarey-bc** [reply]
Any codec with a lossless option, I realize its not ideal for the WebRTC use case but it is a requirement I'm trying to fulfill. I'm implementing vp9 with the lossless option enabled to see if it will work (even if its a really low frame rate, latency is ok). Thanks!

**Reply by danielkoohmarey-bc** [reply]
Following up, I got the lossless vp9 working, im averaging about 3FPS compared to 15FPS with h.264, so there is the expected huge performance hit.
